### PR TITLE
Add template files

### PR DESCRIPTION
**Pull request by samejima-san**
readded the template files from the extensions folders with the desktop/web conditions

### CODE REVIEW

**Comment by samejima-san** [comment]
Need to revert changes in the package.json file, it has information about the test runs I created.

File: packages/modules/create-itwin-viewer/extensions/desktop/intermediate.ts [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/desktop-viewer-itwin-react";
+
+import {
+  ViewerContentToolsProvider,
+  ViewerNavigationToolsProvider,
+  ViewerStatusbarItemsProvider,
+} from "../../../web-viewer-react/node_modules/@itwin/viewer-react/src";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/intermediate.ts:12]: this should be imported from "@itwin/desktop-viewer-react"

File: packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import {
+  TreeWidget,
+  TreeWidgetUiItemsProvider,
+} from "@itwin/tree-widget-react";
+
+import {
+  ViewerContentToolsProvider,
+  ViewerNavigationToolsProvider,
+  ViewerStatusbarItemsProvider,
+} from "../../../web-viewer-react/node_modules/@itwin/viewer-react/src";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts:20]: this should be imported from "@itwin/desktop-viewer-react"

File: packages/modules/create-itwin-viewer/extensions/web/intermediate.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-itwin-react";
+
+import {
+  ViewerContentToolsProvider,
+  ViewerNavigationToolsProvider,
+  ViewerStatusbarItemsProvider,
+} from "../../web-viewer-react/node_modules/@itwin/viewer-react/src";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/intermediate.ts:12]: this should be imported from "@itwin/web-viewer-react"

File: packages/modules/create-itwin-viewer/extensions/web/advanced.ts [hunk 4]
```diff
@@ -11,10 +11,19 @@ import {
   TreeWidget,
   TreeWidgetUiItemsProvider,
 } from "@itwin/tree-widget-react";
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-itwin-react";
 
+import {
+  ViewerContentToolsProvider,
+  ViewerNavigationToolsProvider,
+  ViewerStatusbarItemsProvider,
+} from "../../web-viewer-react/node_modules/@itwin/viewer-react/src";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/advanced.ts:20]: this should be imported from "@itwin/web-viewer-react"

File: packages/modules/create-itwin-viewer/extensions/desktop/intermediate.ts [hunk 5]
```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/desktop-viewer-itwin-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/intermediate.ts:6]: import this from the same package below

File: packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts [hunk 6]
```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/desktop-viewer-itwin-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts:6]: import this from the same package as the others

File: packages/modules/create-itwin-viewer/extensions/web/advanced.ts [hunk 7]
```diff
@@ -11,10 +11,19 @@ import {
   TreeWidget,
   TreeWidgetUiItemsProvider,
 } from "@itwin/tree-widget-react";
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-itwin-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/advanced.ts:14]: import this from the same package as the others

File: packages/modules/create-itwin-viewer/extensions/web/intermediate.ts [hunk 8]
```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-itwin-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/intermediate.ts:6]: import this from the same package as the others

**Review by kckst8 — CHANGES_REQUESTED** [review]

File: packages/modules/create-itwin-viewer/extensions/web/intermediate.ts [hunk 9]
```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "../../web-viewer-react/node_modules/@itwin/web-viewer-react/src";
```
Comment (aruniverse) [inline packages/modules/create-itwin-viewer/extensions/web/intermediate.ts:6]: all these providers should be imported from "@itwin/web-viewer-react" or "@itwin/desktop-viewer-react"

also, there is no need for the "StandardNavigationToolsProvider", we have the "ViewerNavigationToolsProvider" which just wraps those

File: packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+import {
+  PropertyGridManager,
+  PropertyGridUiItemsProvider,
+} from "@itwin/property-grid-react";
+import {
+  TreeWidget,
+  TreeWidgetUiItemsProvider,
+} from "@itwin/tree-widget-react";
+
+import { StandardNavigationToolsProvider } from "../../../web-viewer-react/node_modules/@itwin/desktop-viewer-react/src";
```
Comment (aruniverse) [inline packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts:15]: im not sure why all your imports are relative like this...

**Review by aruniverse — CHANGES_REQUESTED** [review]
need to fix all your imports

**Comment by samejima-san** [comment]
> 
i changed them from the relative imports

File: packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts [hunk 11]
```diff
@@ -0,0 +1,50 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/desktop-viewer-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/advanced.ts:6]: I believe that @aruniverse mentioned this already, but this one is not needed

File: packages/modules/create-itwin-viewer/extensions/desktop/basic.ts [hunk 12]
```diff
@@ -3,5 +3,14 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/basic.ts:5]: there should be no changes to basic

File: packages/modules/create-itwin-viewer/extensions/desktop/intermediate.ts [hunk 13]
```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/desktop-viewer-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/intermediate.ts:6]: I believe that @aruniverse mentioned this already, but this one is not needed

File: packages/modules/create-itwin-viewer/extensions/web/basic.ts [hunk 14]
```diff
@@ -3,5 +3,14 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import { Viewer } from "@itwin/web-viewer-react";
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-react";
+import { ViewerNavigationToolsProvider } from "@itwin/web-viewer-react";
```
Comment (johnnyd710) [inline packages/modules/create-itwin-viewer/extensions/web/basic.ts:8]: You can import all of these at once like so: 
```
import { X, Y, Z } from "...";
```

File: packages/modules/create-itwin-viewer/extensions/web/advanced.ts [hunk 15]
```diff
@@ -11,10 +11,19 @@ import {
   TreeWidget,
   TreeWidgetUiItemsProvider,
 } from "@itwin/tree-widget-react";
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/advanced.ts:14]: I believe that @aruniverse mentioned this already, but this one is not needed

File: packages/modules/create-itwin-viewer/extensions/web/basic.ts [hunk 16]
```diff
@@ -3,5 +3,14 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import { Viewer } from "@itwin/web-viewer-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/basic.ts:6]: this file should have no edits

File: packages/modules/create-itwin-viewer/extensions/web/intermediate.ts [hunk 17]
```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+
+import { StandardNavigationToolsProvider } from "@itwin/web-viewer-react";
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/web/intermediate.ts:6]: I believe that @aruniverse mentioned this already, but this one is not needed

File: packages/modules/create-itwin-viewer/package.json [hunk 18]
```diff
@@ -2,22 +2,24 @@
   "name": "create-itwin-viewer",
   "version": "1.0.0",
   "description": "Initializer for iTwin Viewer apps",
-  "bin": "bin.js",
+  "bin": {
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/package.json:5]: revert these changes

File: packages/modules/create-itwin-viewer/package-lock.json [hunk 19]
```diff
@@ -0,0 +1,310 @@
+{
+  "name": "test",
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/package-lock.json:2]: revert these changes. this is a rush repo so it should not have a package-lock

**Review by kckst8 — CHANGES_REQUESTED** [review]

File: packages/modules/create-itwin-viewer/package.json (hunk 18 above)
Comment (samejima-san) [reply packages/modules/create-itwin-viewer/package.json]: done

File: packages/modules/create-itwin-viewer/package-lock.json (hunk 19 above)
Comment (samejima-san) [reply packages/modules/create-itwin-viewer/package-lock.json]: package-lock.json removed

File: packages/modules/create-itwin-viewer/extensions/web/intermediate.ts (hunk 17 above)
Comment (samejima-san) [reply packages/modules/create-itwin-viewer/extensions/web/intermediate.ts]: done, changed to the viewer navigation tools

File: packages/modules/create-itwin-viewer/extensions/desktop/basic.ts [hunk 20]
```diff
@@ -3,5 +3,8 @@
...
 
+import { Viewer } from "@itwin/desktop-viewer-react";
+
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/extensions/desktop/basic.ts:7]: just revert changes for these files altogether. No need for these extra spaces

File: packages/modules/create-itwin-viewer/package.json [hunk 21]
```diff
@@ -10,14 +10,14 @@
   },
   "repository": {
     "type": "git",
-    "url": "https://github.com/iTwin/viewer.git"
+    "url": "git+https://github.com/iTwin/viewer.git"
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/package.json:13]: what is this change for?

File: packages/modules/create-itwin-viewer/package.json [hunk 22]
```diff
@@ -10,14 +10,14 @@
...
   },
   "keywords": [
     "iTwin",
     "Viewer",
     "iTwin.js"
   ],
-  "author": {
+  "author":   {
```
Comment (kckst8) [inline packages/modules/create-itwin-viewer/package.json:20]: please remove the added space

**Review by kckst8 — CHANGES_REQUESTED** [review]

File: packages/modules/create-itwin-viewer/package.json (hunk 21 above)
Comment (samejima-san) [reply packages/modules/create-itwin-viewer/package.json]: it wasn't a manual change, i just reverted it though.

File: packages/modules/create-itwin-viewer/extensions/desktop/basic.ts (hunk 20 above)
Comment (samejima-san) [reply packages/modules/create-itwin-viewer/extensions/desktop/basic.ts:7]: do you mean the empty lines between?

**Review by kckst8 — APPROVED** [review]
